doc/ubuntu_release_process.md

```diff
@@ -203,6 +203,74 @@ We have daily packaging recipes that upload to the [daily ppa](https://code.laun
   * [bionic](https://code.launchpad.net/~cloud-init-dev/+recipe/cloud-init-daily-bionic)
   * [devel](https://code.launchpad.net/~cloud-init-dev/+recipe/cloud-init-daily-devel)
 
+### When the daily recipe build fails ###
+
+The daily recipe for each release checks out master and then merges the ubuntu/$release
+and builds the package from there.  From time to time, the patches in the
+ubuntu/$release branch need to be refreshed/updated as upstream/master changes.
+
+This is typically done during the new-upstream-release tool process, however,
+new commits to master can break the patches.  In particular, the
+ubuntu-advantage refactor is reverted on bionic and xenial; this patch needs
+to be refreshed whenever changes to cloud-init touch:
+
+ * cloudinit/config/cc_ubuntu_advantage.py
+ * cloudinit/config/tests/test_ubuntu_advantage.py
+
+At this point if daily builds are failing, this will fail when quilt
+cannot apply the patch and put you into a subshell and ask you to
+fix and then quit refresh.  The source of the issue is that the release
+branch needs the upstream changes to ensure the patch still applies.
+Do the following to fix the patch
+
+    $ new-upstream-snapshot -v
+    $ (refresh-fix)(hostname) cloud-init % quilt push -f
+    $ (refresh-fix)(hostname) cloud-init % git checkout 528366820bb48c13957d0c58afc2a46a3ba84bef cloudinit/config/tests/test_ubuntu_advantage.py
```

Comment:
Maybe add the comment `# Either manually change the files that need patching, or checkout the specific commitish for the file desired content state

Reply:
Yes, +1